src/components/Timelog/TimeEntryForm/TimeEntryForm.jsx

```diff
@@ -320,10 +321,27 @@ const TimeEntryForm = props => {
       }
     }
 
+    // if sum of reported tangible hours + volunteer time is greater than the previous personal best max hours
+    // we update the personal best max hours
+    let should_update_badge = false
+    if (parseFloat(userProfile.tangibleHoursReportedThisWeek) + volunteerTime > userProfile.personalBestMaxHrs) {
```

Comment:
This is what I do to simulate beating my personal max:
0. Set up: Made the cron job to run every 1 minute so I don't need to wait until the end of the week to see if your feature works
1. Start timer
2. Wait 1 minute
3. Stop Timer
4. Edit the time to be 2 hours to beat this week record

**Issue:**
This does not update the `user.personalBestMaxHrs`. Because this variable is only updated in `updateHoursByCategory()` function but not in `editHoursByCategory()`.
Shouldn’t this code also be placed in `editHoursByCategory()`?

Cron job code that I edited:
<img width="563" alt="Screenshot 2023-07-18 at 4 47 29 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/67333705/c89e4e03-cf13-43e5-b6fc-4d5addb26b92">


Confirmed that cron is awarding new badges every 1 minute (don't need til the end of the week)
<img width="353" alt="Screenshot 2023-07-18 at 4 47 39 PM" src="https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/67333705/66999cd8-2b6a-44ae-b8e8-31e24e3267c1">